TGM/src/main/java/network/warzone/tgm/command/CycleCommands.java

```diff
@@ -530,15 +539,15 @@ public static void stats(final CommandContext cmd, CommandSender sender) {
     public static void viewStats(CommandSender sender, String target) {
         Player targetPlayer = Bukkit.getServer().getPlayer(target);
         if (targetPlayer == null) {
-            sender.sendMessage(ChatColor.RED + "Unable to find online player " + ChatColor.YELLOW + target);
+            sender.sendMessage(ChatColor.DARK_RED + "Unable to find online player " + ChatColor.RED + target);
```

Comment:
Idk about this, dark red is pretty awful.

Reply:
I agree, error messages should follow the Warzone standard of red for text and dark red for the highlighted stuff, or in this case yellow because it's a player